### ISSUE TITLE
Incorrect role is defined for "More Action...(Shift+F9)"button as menuitem: A11y_VS Code_Settings_AI4D

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#214222]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags
#A11yTCS; #A11ySev2; #WCAG4.1.2; #BM-VisualStudioCodeClient-Win32-Jan2024 ; #DesktopApp; #Visual Studio Code Client; #A11yMAS; #Name Role Value; #Win32; #AI4D;

### Environment Details:
Application Name: Visual Studio Code Client
Visual Studio Code: Version 1.89.1(User setup)

### Repro Steps:

1. Install VS Code
2. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
3. Open the Command Palette via the keyboard command Ctrl+Shift+P (or Command+Shift+P for MacOS)
4. Type "Open User Settings" and select the option from the dropdown list called "Preferences: Open User Settings") and press enter
5. In the search box, type "Azure Machine Learning"
6. Tab until the list of settings is active
7. Use the up/down arrows to navigate between settings
8. Tab till "More Action...shift+F9" button and Verify whether the correct role is defined for "More Action...(Shift+F9)"button or not.

### Actual Result:
Incorrect role is defined for "More Action...(Shift+F9)"button as menuitem.

### Expected Result:
Correct role should be defined for "More Action...(Shift+F9)" as button instead of menuitem.

### User Impact:
Screen reader users will face difficulty in accessing the control if the correct control type is not defined for the controls.

### Attachment:
![image](https://github.com/microsoft/vscode/assets/98530069/f69814f2-31a3-448b-b3cf-271e7dbcfa5b)

**Comment by meganrogge** [comment]
@rzhao271 let me know if you want help here

**Comment by rzhao271** [comment]
In the code, the aria role of the element is actually "button".
However, the element also has aria-haspopup and aria-expanded set in [dropdownActionViewItem.ts](https://github.com/microsoft/vscode/blob/f1808e12bdaa0638ff5b8fc10e867571c1d614c1/src/vs/base/browser/ui/dropdown/dropdownActionViewItem.ts#L93), because clicking on the button does reveal a menu.
Therefore, NVDA reads `More Actions... (Shift+F9)  menu button  collapsed  subMenu`

**Comment by rzhao271** [comment]
@kupatkar99 It looks like the aria role is already defined as button. I'm wondering why the software identifies the element as a menu item? NVDA correctly calls the element a menu button because clicking on it shows a menu.

![aria role is button](https://github.com/user-attachments/assets/4135d92e-f996-4b5a-bef0-214692738090)

**Comment by kupatkar99** [comment]
Closing the bug as non-blocking issue as screen reader user will be able to understand the purpose of the control. hence closing the bug.
